test/test_type_hints.py

```diff
@@ -170,6 +170,20 @@ def test_doc_examples(self):
             except subprocess.CalledProcessError as e:
                 raise AssertionError("mypy failed.  Look above this error for mypy's output.")
 
+    def test_type_hint_examples(self):
+        """
+        Runs mypy over all the test examples present in
+        type_hint_tests directory
+        """
+
+    examples = os.listdir(os.getcwd() + '/test/type_hint_tests/')
```

Comment:
This is assuming that we're running the test from the root pytorch folder, right?

It might be better to be current-directory-agnostic:
```
test_path = os.path.dirname(os.path.realpath(__file__))
examples_folder = os.join(test_path, "type_hint_tests")
examples = os.listdir
```

Reply:
Done